### DISCUSSION TITLE
September Community call

### DISCUSSION THREAD

**Discussion by eddiejaoude** [EddieHubCommunity/support#2539] [Community Call]
Welcome to the EddieHub Community Call for September!

Let's ask questions and discuss things, here is how:

**Start a new thread, and we can get responses on both the call and inside this github Discussion**

You can drop an intro in the introduction thread below too

**Comment by eddiejaoude** [comment]
Give yourself an intro and shoutout 👇 (you can include links)

**Comment by naomi-lgbt** [comment]
We need some positivity this week: 

Use this comment to share something good! It can be something you achieved, something you built, or something that just happened. 🙂

**Reply by eddiejaoude** [reply]
Yes ☝️  I got 14k follows on Twitter today 🥳

**Reply by iason9** [reply]
> Yes ☝️ I got 14k follows on Twitter today 🥳

Congrats!

**Reply by atharvashirdhankar** [reply]
Congrats Eddie🎉

**Comment by naomi-lgbt** [comment]
I'm doing a workshop so have to miss the call - we should use this comment to share something cool you learned/heard in today's community call!

**Comment by Vyvy-vi** [comment]
Some more Positivity! -
Shoutout someone from the EddieHub community that added value or helped you out this month, or shared their wisdom during the call.
You could maybe write something like this-
```
- Thanks for helping me out for ... and sharing your thoughts on ...
- I really appreciate what you did for the community this week, and all your work on ... project
- This is something I noticed that you could maybe work on or could have handled ... differently
```

**Reply by eddiejaoude** [reply]
Love this 💥

**Comment by SUCHITRAGIRI** [comment]
Today's call is really insightful !
My question was:
As I know, Data Structure and Algorithm is a crucial part for cracking  most of the company interviews, but is it not possible to skip it (if we are not interested on it) and spend our timing on our tech stacks?
Why it is so much important even for dev roles!!

**Reply by eddiejaoude** [reply]
Not sure people will write their answers now after the call, I can't remember everyones responses either 🤦‍♂️  But hopefully you remember and got value from people's answers 🤓
